db/1.9/modules/ROOT/pages/database-configure-data-types.adoc

```diff
@@ -40,7 +40,57 @@ In the XML editor, the `<db:parameter-types>` configuration looks like this:
 
 == Configure the Column Types Field in Studio
 
-Each connection provider uses exclusive and common parameters, such as Derby and Oracle. A child element of the connection provider element defines custom data types parameters you can use when connected to a particular provider. The following example shows how to configure the Column Types field in the Derby global configuration connection element:
+Each connection provider uses exclusive and common parameters, such as Derby and Oracle. A child element of the connection provider element defines custom data types parameters you can use when connected to a particular provider.
```

Comment:
Can we remove the first sentence? The term "exclusive" doesn't sound quite right but I can't think of what to replace it with. Up to you.